CHANGES.rst

```diff
@@ -507,6 +511,9 @@ astropy.convolution
 
 - Fixed a bug [#9168] where having a kernel defined using unitless astropy
   quantity objects would result in a crash [#9300]
+- Fixed a bug in ``discretize_oversample_1D/2D()`` from ``astropy.convolution.utils``,
```

Comment:
this shouldn't be here?

Reply:
Weird, much be some rebase problem.

Reply:
yes, I was moving moving around the entries, git might got confused....